src/config/contracts.mainnet.json

```diff
@@ -0,0 +1,15 @@
+{
+  "rns": "0xcb868aeabd31e2b66f74e9a55cf064abb31a4ad5",
+  "registrar": "0x5269f5bc51cdd8aa62755c97229b7eeddd8e69a6",
+  "reverseRegistrar": "0xd25c3f94a743b93ecffecbe691beea51c3c2d9d1",
+  "publicResolver": "0x4efd25e3d348f8f25a14fb7655fba6f72edfe93a",
+  "nameResolver": "0x4b1a11bf6723e60b9d2e02aa3ece34e24bde77d9",
+  "multiChainResolver": "0x99a12be4C89CbF6CFD11d1F2c029904a7B644368",
+  "rif": "0x2acc95758f8b5f583470ba265eb685a8f45fc9d5",
+  "fifsRegistrar": "0x779195c53cc7c1a33bd2eea5f63f2c1da8798d61",
+  "fifsAddrRegistrar": "0xd9c79ced86ecf49f5e4a973594634c83197c35ab",
+  "rskOwner": "0x45d3e4fb311982a06ba52359d44cb4f5980e0ef1",
+  "renewer": "0x7a9872a7615c475b62a62b8f6e491077fb05f663",
+  "stringResolver": "",
```

Comment:
What `stringResolver` is used for? why isn't it set?

Reply:
I'm not sure how is used `stringResolver` or even if it is use at all, but I could not find the address of this one on [mainnet DevPortal](https://dev.rootstock.io/rif/rns/mainnet/).

Reply:
it looks like it's used [here](https://github.com/rsksmart/rns-manager-react/blob/71a8df4e225f96e6fe6c353e67d93136b2b33604/src/app/tabs/newAdmin/resolver/operations.js#L51)

Reply:
Setting `stringResolver` to address Zero, according to @LordCheta